jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java

```diff
@@ -131,7 +131,7 @@ public JdrReport collect() throws OperationFailedException {
                 ROOT_LOGGER.debugf(message);
                 skips.append(message);
                 PrintWriter pw = new PrintWriter(new StringWriter());
-                t.printStackTrace(pw);
+                ROOT_LOGGER.error(t.getLocalizedMessage());
```

Comment:
Your change is breaking original behavior, you are appending nothing. This change should be reverted

Reply:
gosh, sorry, I did not noticied the PrintWriter manipulation. I'll fix that right away.